### ISSUE TITLE
Puppet failing with apt key mismatch

### ISSUE THREAD

**Issue by mdechiaro** [puppetlabs/puppet_operational_dashboards#191]
## Describe the Bug
The `puppet-grafana` module was recently updated to use a new key, but the latest release of this module, `2.0.0` still sources the old key. This is causing Puppet to fail. See https://github.com/voxpupuli/puppet-grafana/pull/335 for the change. Here is the error when Puppet runs after upgrading to Grafana 13.0.1. It seems this change needs to be released to fix https://github.com/puppetlabs/puppet_operational_dashboards/commit/98074bf3

```
2023-09-25 09:05:37 -0500 /Stage[main]/Influxdb/Apt::Source[influxdb2]/Apt::Setting[list-influxdb2]/File[/etc/apt/sources.list.d/influxdb2.list]/ensure (notice): defined content as
'{sha256}d7473813afec7171e00ae6fe0db275ab4e652ebccb88025a659cb808d258f850'
2023-09-25 09:05:37 -0500 /Stage[main]/Telegraf::Install/Apt::Source[influxdata]/Apt::Setting[list-influxdata]/File[/etc/apt/sources.list.d/influxdata.list]/ensure (notice): defined content
as '{sha256}c4f446859eb6dd3ff34a0acf62676fc1a4e9fc2f68209e74a686f312d878506a'
2023-09-25 09:05:37 -0500 Puppet (err): Could not set 'present' on ensure: The id in your manifest 0E22EB88E39E12277A7760AE9E439B102CF3C0C6 and the fingerprint from content/source don't
match. Check for an error in the id and content/source is legitimate. (file: /etc/puppetlabs/code/environments/production/modules/apt/manifests/key.pp, line: 54)
2023-09-25 09:05:37 -0500 Puppet (err): Could not set 'present' on ensure: The id in your manifest 0E22EB88E39E12277A7760AE9E439B102CF3C0C6 and the fingerprint from content/source don't
match. Check for an error in the id and content/source is legitimate. (file: /etc/puppetlabs/code/environments/production/modules/apt/manifests/key.pp, line: 54)
Wrapped exception:
The id in your manifest 0E22EB88E39E12277A7760AE9E439B102CF3C0C6 and the fingerprint from content/source don't match. Check for an error in the id and content/source is legitimate.
2023-09-25 09:05:37 -0500 /Stage[main]/Grafana::Install/Apt::Source[grafana]/Apt::Key[Add key: 0E22EB88E39E12277A7760AE9E439B102CF3C0C6 from Apt::Source grafana]/Apt_key[Add key:
0E22EB88E39E12277A7760AE9E439B102CF3C0C6 from Apt::Source grafana]/ensure (err): change from 'absent' to 'present' failed: Could not set 'present' on ensure: The id in your manifest
0E22EB88E39E12277A7760AE9E439B102CF3C0C6 and the fingerprint from content/source don't match. Check for an error in the id and content/source is legitimate. (file:
/etc/puppetlabs/code/environments/production/modules/apt/manifests/key.pp, line: 54)
2023-09-25 09:05:37 -0500 /Stage[main]/Grafana::Install/Apt::Source[grafana]/Apt::Key[Add key: 0E22EB88E39E12277A7760AE9E439B102CF3C0C6 from Apt::Source grafana]/Anchor[apt_key
0E22EB88E39E12277A7760AE9E439B102CF3C0C6 present] (notice): Dependency Apt_key[Add key: 0E22EB88E39E12277A7760AE9E439B102CF3C0C6 from Apt::Source grafana] has failures: true
2023-09-25 09:05:37 -0500 /Stage[main]/Grafana::Install/Apt::Source[grafana]/Apt::Key[Add key: 0E22EB88E39E12277A7760AE9E439B102CF3C0C6 from Apt::Source grafana]/Anchor[apt_key
0E22EB88E39E12277A7760AE9E439B102CF3C0C6 present] (warning): Skipping because of failed dependencies
2023-09-25 09:05:37 -0500 /Stage[main]/Grafana::Install/Apt::Source[grafana]/Apt::Setting[list-grafana]/File[/etc/apt/sources.list.d/grafana.list] (warning): Skipping because of failed
dependencies
```

## Expected Behavior
Puppet should run without error

## Steps to Reproduce
1. Upgrade puppet-grafana to latest release, `13.0.1`
2. Puppet will fail

## Environment
 - Version 7-25-01
 - Platform Ubuntu 22.04

**Comment by MartyEwings** [comment]
hi there

I believe we have this resolved in https://github.com/puppetlabs/puppet_operational_dashboards/pull/190 but will need to do a module release later today

**Comment by mdechiaro** [comment]
Appreciate the fast turn-around and look forward to the release.

**Comment by m0dular** [comment]
Should be resolved in 2.1.0, let us know if you see the issue reoccur.
